Tutorial: Return Cases from 100 Years Ago Today with the CAP API

lkfitz opened the pull request:
Tutorial: Return Cases from 100 Years Ago Today with the CAP API

lkfitz committed:
Tutorial: Return Cases from 100 Years Ago Today with the CAP API

Tutorial: Return Cases from 100 Years Ago Today with the CAP API

bensteinberg commented on app/_posts/2019-07-31-tutorial-return-cases-from-100-years-ago-today-with-the-cap-api.md:
```diff
@@ -0,0 +1,58 @@
+title: Tutorial: Return Cases from 100 Years Ago Today with the CAP API
+author: kelly-fitzpatrick
+---
+The [Caselaw Access Project API](https://case.law/api/) offers a way to view the corpus of U.S. case law. This tutorial will review **run a CAP API call to return all cases decided 100 years ago today in your command line**.
```
I think this sentence is missing a word or two.

bensteinberg commented on app/_posts/2019-07-31-tutorial-return-cases-from-100-years-ago-today-with-the-cap-api.md:
```diff
@@ -0,0 +1,58 @@
+---
+title: Tutorial: Return Cases from 100 Years Ago Today with the CAP API
```
Wrap the title in quotation marks when it contains a colon.

lkfitz committed:
Update 2019-07-31-tutorial-return-cases-from-100-years-ago-today-with-the-cap-api.md

lkfitz committed:
Update 2019-07-31-tutorial-return-cases-from-100-years-ago-today-with-the-cap-api.md

lkfitz committed:
Update 2019-07-31-tutorial-return-cases-from-100-years-ago-today-with-the-cap-api.md

Fixing misc missing words; tweaks.

bensteinberg committed:
Formatting and wording tweaks

The only notable changes here are the use of `{% highlight %}` for code blocks and the replacement of "endpoints" with "parameters". A few params got backticks for inline fixed-width formatting.
